LibraBFT/Abstract/BFT.agda

```diff
@@ -6,11 +6,479 @@
 open import LibraBFT.Prelude
 open import LibraBFT.Lemmas
 open import LibraBFT.Abstract.Types
+open import LibraBFT.Base.PKCS
 
-module LibraBFT.Abstract.BFT where
+
+module LibraBFT.Abstract.BFT
+  (authorsN  : ℕ)
+  (bizF      : ℕ)
+  (isBFT     : authorsN ≥ suc (3 * bizF))
+  (getPubKey : Fin authorsN → PK)
+
+
+
+ where
 
   -- This is a utility function to make it easy to provide the bft-assumption
```

Comment:
Please update the comment to make clear that the purpose of the module is `bft-lemma` at the end.

Reply:
Done.